icu4j/main/common_tests/src/test/java/com/ibm/icu/dev/test/number/NumberParserTest.java

```diff
@@ -199,13 +197,13 @@ public void testLocaleFi() {
 
         ParsedNumber resultObject = new ParsedNumber();
         parser.parse("epäluku", false, resultObject);
-        assertTrue(resultObject.success());
-        assertEquals(Double.NaN, resultObject.getNumber().doubleValue(), 0.0);
+        assertTrue("Parsing locale fi spellout number", resultObject.success());
+        Assert.assertEquals(Double.NaN, resultObject.getNumber().doubleValue(), 0.0);
```

Comment:
Inconsistent style: `assertTrue` but `Assert.assertEquals`.
Either everything is static import, or nothing is.

---

This should maybe be a bigger cleanup, because `TestFmwk` defines some asserts (for example `assertTrue`).
So when you see `assertTrue(...)` it is hard to tell if it is the `TestFmwk` one or the `JUnit` one. (why should we even have one in `TestFmwk`)